### ISSUE TITLE
[SF][A11Y][Combobox]: Announce Subsections with Group Item by Screen Reader

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#5244]
Hi Colleague,

We used ui5-combobox in OMP project for filter. Recently, QAS team raised a ticket about SR announcement. Could you pls take a look? Below is the details:
Visually we can see there are 2 subsections – Assignment & hiring status, unable to check the JAWS announcement whether they are associated with subsections or not Eg- “Assignment status Draft/ published/Ongoing”
![image](https://user-images.githubusercontent.com/78520316/169187812-bad64c21-ebeb-4279-997a-9288c0c32ee1.png)

From ui5-wc playground(https://sap.github.io/ui5-webcomponents/playground/components/ComboBox/), the combobox with group item doesn't announce the specific group of selected options.
![image](https://user-images.githubusercontent.com/78520316/169188145-b88c3298-9a25-4862-ba87-086d450d5dde.png)

### **Proposed Solution**
When focus on the selected item,  Subsection should be announced to tell user which group this item belong to. 

FYI:
ui5-wc version: 1.3.1

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

Best Regards,
Josephine Su

**Comment by LidiyaGeorgieva** [comment]
Hello @SAP/ui5-webcomponents-topic-rl  take a look of this ACC issue.

**Comment by elenastoyanovaa** [comment]
Hello,

The concept of announcing the group in which the focused list item belong was not considered in the current screen reader specification. As we follow the current implementation for UI5 List and ComboBox the functionality is not present in the ui5-combobox, because it was not specified for the UI5 ComboBox and UI5 List as well. You can check the samples here:
https://sapui5.hana.ondemand.com/#/entity/sap.m.List/sample/sap.m.sample.ListGrouping
https://sapui5.hana.ondemand.com/#/entity/sap.m.ComboBox/sample/sap.m.sample.ComboBoxGrouping

I would consider this as a feature request and we will evaluate the desired behavior and get back to you.

Kind Regards,
Elena
